guides/process/scheduling/how_to_timesheet.md

```diff
@@ -44,6 +44,20 @@ You should allocate your time to a project when you’ve spent 2 hours or more.
 
 We use Kimble to complete timesheets. All resources on how to complete and approve timesheets are in the [Knowledge base](https://sites.google.com/madetech.com/signpost/home/kimble-resources?authuser=0).
 
+### Why we need you to do timesheets
+
+Timesheet data directly drives our client invoicing. If you don't timesheet, the business can't invoice and we lose revenue which has an impact on all of us.
+Timesheet data also creates wider business-critical data about utilisation, client investments and time spent on other activities that's needed for reporting and planning.
+
+### Why we need you to do timesheets weekly
+Whilst we invoice our clients monthly it's vitally important that the data driving these invoices is accurate as clients do actually check, after all nobody likes being overcharged for anything. Weekly timesheets ensure we capture time spent while it's fresh in our memories.
+
+Doing them weekly provides immediate visibility of time spent for line and delivery managers to create better quality project level conversations about time spent and weekly-level understanding of budget burn-down for client conversations
+
+This allows us to update our monthly revenue forecast in real time: helping us understand whether we're on course to meet projected income due to things like sick leave, or last minute changes to time allocation, and readjust targets and cashflow forecasts as we go.
+
+It’s important that the data is accurate and complete at the end of every week. That means midday on Friday, not 9am the following Monday morning. (Ops and Finance really do work on Friday afternoons to make sure everything is in so the week can be closed off!)
```

Comment:
A final wording amend:
'It's important that the data is accurate and complete at the end of every week. A number of business processes - including scheduling, revenue forecasting and cashflow - need accurate data to drive them on a Monday. Any timesheet submitted after 12pm on a Friday impacts our ability to generate good quality data and causes significant extra work for the Ops and Finance teams. Please do them on time!'

Reply:
Mostly updated